### PR TITLE
[Jsontracer] Fix RegExp in index.html

### PR DESCRIPTION
**Pull request by wnsdud4197**
this will fix RegExp in Jsontracer/index.html, which was applied incorrectly

ONE-vscode-DCO-1.0-Signed-off-by: Junyeong Yang <wnsdud4197@naver.com>

### CODE REVIEW

**Comment by wnsdud4197** [comment]
from draft #126
from issue #94

**Review by parkjisu6239 — DISMISSED** [review]

File: media/Jsontracer/index.html [hunk 1]
```diff
@@ -49,11 +49,19 @@
 -->
 <head>
   <meta charset="utf-8">
+  <!--
+    Use a content security policy to only allow loading images from https or from our extension directory,
```
Comment (llFreetimell) [inline media/Jsontracer/index.html:53]: Is there any case that we loads images from https?

Comment (wnsdud4197) [reply media/Jsontracer/index.html:53]: we don't load images. so i will change `img-src 'none'` and comment

**Review by parkjisu6239 — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**